CMakeLists.txt

```diff
@@ -605,7 +624,8 @@ init merge-subtree restore show
 stage status switch whatchanged)
 
 #Creating symlinks
-
+#In CI symlinks don't work harlinks do.
```

Comment:
It's not about CI. The `Makefile` wants to make hardlinks (with a `d` BTW), and so should the CMake-based build.

Reply:
Okay, switching the build to use only hardlinks from now on.